hooks/useObservations.ts

```diff
@@ -392,24 +390,28 @@ export const ObservationDocument = `
   }
 }
     ${EverythingFragmentDoc}`;
-export const useObservationQuery = <TData = ObservationQuery, TError = unknown>(variables: ObservationQueryVariables, options?: UseQueryOptions<ObservationQuery, TError, TData>) =>
-  useQuery<ObservationQuery, TError, TData>(['observation', variables], useFetch<ObservationQuery, ObservationQueryVariables>(ObservationDocument).bind(null, variables), options);
+
 export const ObservationsDocument = `
     query observations($center: String!, $startDate: String!, $endDate: String!) {
   getObservationList(centerId: $center, startDate: $startDate, endDate: $endDate) {
     ...overview
   }
 }
     ${OverviewFragmentDoc}`;
-export const useObservationsQuery = <TData = ObservationsQuery, TError = unknown>(
-  variables: ObservationsQueryVariables,
-  options?: UseQueryOptions<ObservationsQuery, TError, TData>,
-) =>
-  useQuery<ObservationsQuery, TError, TData>(
-    ['observations', variables],
-    useFetch<ObservationsQuery, ObservationsQueryVariables>(ObservationsDocument).bind(null, variables),
-    options,
-  );
 
-// No matter what span of time we're looking at, we will never query over more than 3 months of observations
-export const MAXIMUM_OBSERVATIONS_LOOKBACK_WINDOW: Duration = {months: -3};
+// These are unused - can they be deleted?
```

Comment:
@stevekuznetsov thoughts?

Reply:
Autogenerated and can be removed I think